x-pack/plugins/session_view/public/test/index.tsx

```diff
@@ -68,8 +67,8 @@ const AppRootProvider = memo<{
   history: History;
   coreStart: CoreStart;
   children: ReactNode | ReactNode[];
-}>(({ history, coreStart: { http, notifications, uiSettings, application }, children }) => {
-  const isDarkMode = useObservable<boolean>(uiSettings.get$('theme:darkMode'));
+}>(({ history, coreStart: { http, notifications, theme, application }, children }) => {
+  const isDarkMode = useMemo(() => theme.getTheme().darkMode, [theme]);
```

Comment:
**nit(non-blocking):** same as another comment, I think the memoization here is not necessary.